fdbclient/include/fdbclient/BlobCipher.h

```diff
@@ -353,7 +404,8 @@ struct BlobCipherKeyIdCache : ReferenceCounted<BlobCipherKeyIdCache> {
 	                                             const uint8_t* baseCipher,
 	                                             int baseCipherLen,
 	                                             const int64_t refreshAt,
-	                                             const int64_t expireAt);
+	                                             const int64_t expireAt,
+	                                             bool* newCipherKeyCreated);
```

Comment:
why do we need `newCipherKeyCreated`, the routine internally determines if a `new cipher` needs to be created OR an existing `key` is getting updated

```
if (latestCipherKey.isValid() && latestCipherKey->getBaseCipherId() == baseCipherId) {
		if (memcmp(latestCipherKey->rawBaseCipher(), baseCipher, baseCipherLen) == 0) {
```

Reply:
Something from my previous version that I didn't cleanup. Let me do so.

Reply:
It is an output param instead of an input param. It indicates whether a new cipher key is created and is used to maintain the cache size count.

Reply:
My point is the current function internally governs/decides if a new-cipher needs to be created, why to have input parameter for it? (also it might not be accurate).

Also for metrics point of new, `new cipher` count can be maintained by incrementing counter in the `initKey` or ctor of `BlobCipherKey` for accurate counts..